README.md

```diff
@@ -11,24 +11,49 @@ This project is for OSIS which integrates [Scality RING](https://www.scality.com
 
 ## Running the Application
 
-### Command to run Docker Image
-A user needs to be logged into docker using URL: registry.scality.com.
-```sh
-    $ docker login -u <username in registry.scality.com> registry.scality.com
-```
-Once logged images can be pulled and run locally from either dev namespace using the hash or production namespace.
+### Commands to run Docker Image
+1. Docker images can be pulled locally from either dev namespace using the hash or production namespace.
+
+    Dev:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
+    ```
+    Production image:
+    ```sh
+        $ docker pull registry.scality.com/vmware-ose-scality/vmware-ose-scality:<tag>
+    ```
+1. Generate a self-signed SSL certificate and store it in a binary PKCS#12 format file with extension `.p12` file. (Refer [here](#To-generate-PKCS12-file-for-self-signed-SSL-certificate))
+
+1. Create an environment variables file `application.properties`
+    - Sample `application.properties` file can be found [here](src/main/resources/application.properties).
+    - Update `server.ssl.key-store-password` and `server.ssl.key-alias` with key-store password and key-store alias of the `.p12` file respectively.
+    
+1. To run the docker image locally from either dev namespace using the hash or production namespace with the environment file `application.properties`
+
+    Dev:
+    ```sh
+        $ docker run --env-file application.properties -it -p8443:8443 -v <absolute_path_to_.p12_file>:/app/lib/osis.p12:z registry.scality.com/vmware-ose-scality-dev/vmware-ose-scality:<short SHA-1 commit hash>
```

Comment:
Missing space after `-p` and also I think we should use the `--mount` flag instead of `-v`. From here https://docs.docker.com/storage/volumes/#choose-the--v-or---mount-flag it seems the mount flag is pushed more.

Reply:
Covered this in e3ae684ae588760d0b37f12fe614eda2c282a644